app/src/main/java/org/mozilla/fenix/home/recentsyncedtabs/RecentSyncedTabFeature.kt

```diff
@@ -107,6 +113,17 @@ class RecentSyncedTabFeature(
             RecentSyncedTabs.latestSyncedTabIsStale.add()
         }
     }
+
+    // SYNC_UNAVAILABLE is sent even though displaySyncedTabs is never called when an account
```

Comment:
Could use a description of what fatal actually means beyond the individual error types